core/math/transform.cpp

```diff
@@ -113,6 +113,41 @@ void Transform::set_look_at( const Vector3& p_eye, const Vector3& p_target, cons
 
 }
 
+Transform Transform::looking_at_dir( const Vector3& p_target, const Vector3& p_up ) const {
+	Transform t = *this;
+	t.set_look_at_dir(p_target,p_up);
+	return t;
+}
+
+void Transform::set_look_at_dir(const Vector3& p_target, const Vector3& p_up ) {
+
+	// Reference: MESA source code
+	Vector3 v_x, v_y, v_z;
+
+	/* Make rotation matrix */
+
+	/* Z vector */
+	v_z = -p_target;
```

Comment:
This is inconsistent,  with the opengl where -z is positive, from the perspective of time I'm not sure if this is good